README.md

```diff
@@ -65,6 +65,8 @@ This version of Meteor Up is powered by [Docker](http://www.docker.com/), making
     cd .deploy
     mup init
 
+***WARNING: Windows users need to use `mup.cmd` instead. As `mup` will result in unexpected behavior.
```

Comment:
There is a warning 7 lines down, but your is in a better place. Please remove the old one.